liquibase-core/src/main/java/liquibase/sqlgenerator/core/util/MSSQLUtil.java

```diff
@@ -0,0 +1,67 @@
+package liquibase.sqlgenerator.core.util;
+
+import liquibase.sql.Sql;
+import liquibase.sql.UnparsedSql;
+import liquibase.structure.core.Relation;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * MSSQL Util for generator to handle SET methods for now.
+ * Additional utility methods can be added here
+ */
+public class MSSQLUtil {
```

Comment:
I would recommend this class to point that it's mssql util for SQLGenerators.
For example: SqlGeneratorMssqlUtil or something like that

Reply:
Done!

Reply:
Changed to SqlGeneratorMSSQLUtil.